app/views/employees/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Check In</title>
+</head>
+<body>
+  <h1>Employees Index</h1>
+  <p>
+    <label for="text">#ID EMPLOYEE</label>
+    <input type="text">
+    <button>Check In</button> <br>
+
+    <button><%= link_to "Admin view", administrators_path%></button>
```

Comment:
you may add some button classes into link_to helper instead of wrapping it with a button tag